scripts/kind.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -o errexit
+export MY_IP=$(ip -o route get to 8.8.8.8 | sed -n 's/.*src \([0-9.]\+\).*/\1/p')
+# desired cluster name; default is "kind"
```

Comment:
This will not work on MacOS.  Are you just trying to get a `localhost` address from this command?

Reply:
Stashing here for future reference.  I think this might work on both MacOS and most Linux distros but I'm still not sure why it's better than just using the loopback address `127.0.0.1` to reach a local Kind cluster.

```ifconfig | grep 'inet ' | grep -v '127.0.0.1' | awk '{print $2}'```

Reply:
You are right, 127.0.0.1 is best, this was in the script I copied from another project, at home I run Kind on a workstation and access it from my laptop, thus the need to listen on the machine public ip